markdown/src/androidTest/java/it/niedermann/android/markdown/MarkdownUtilTest.java

```diff
@@ -234,55 +234,55 @@ public void testTogglePunctuation() {
 
         // Remove containing punctuation
         builder = new SpannableStringBuilder("Lorem *ipsum* dolor sit amet.");
-        assertEquals(27, MarkdownUtil.togglePunctuation(builder, 0, 29, "*"));
+        assertEquals(11, MarkdownUtil.togglePunctuation(builder, 0, 29, "*"));
         assertEquals("Lorem ipsum dolor sit amet.", builder.toString());
 
         // Remove multiple containing punctuations
         builder = new SpannableStringBuilder("Lorem *ipsum* dolor *sit* amet.");
-        assertEquals(27, MarkdownUtil.togglePunctuation(builder, 0, 31, "*"));
+        assertEquals(21, MarkdownUtil.togglePunctuation(builder, 0, 31, "*"));
         assertEquals("Lorem ipsum dolor sit amet.", builder.toString());
 
         // Special use-case: toggle from italic to bold and back
 
         // TODO Toggle italic on bold text
```

Comment:
Please remove the `TODO` in front of the comments, they are no longer `TODO` since you have implemented them in this PR :slightly_smiling_face: